app/src/forms/admin/routes.js

```diff
@@ -18,22 +18,175 @@ routes.use(currentUser);
 //
 // Forms
 //
+
+/**
+ * @openapi
+ * /admin/forms:
+ *  get:
+ *    tags:
+ *      - Admin
+ *    summary: List all forms in the system
+ *    description: This endpoint will fetch all forms under the admin access.
+ *    security:
+ *      - openId: []
+ *    responses:
+ *      '200':
+ *        description: Success
+ *        content:
+ *          application/json:
+ *            schema:
+ *              $ref: '#/components/responses/responseBody/AdminsFormGetEx'
+ *      '403':
+ *        $ref: '#/components/responses/Error/AccessDenied'
+ *      '5XX':
+ *        $ref: '#/components/responses/Error/UnExpected'
```

Comment:
Do we want to call this 500 instead of 5XX? There are many 5XX codes that are not "Unexpected". It's really only 500 that's unexpected because it's a server error. New developers might also be confused that there's actually a 5XX error?

We'll also have things like 503 when the OpenShift cluster is really busy, but we don't need to document those... although they should be "expected".

Reply:
Or do we want to leave it as 5XX so devs handle all 5XX errors? @jasonchung1871?

Reply:
The 5XX is for swagger is used to denote unknown or unexpected error.